api/samples/bbbuf.c

```diff
@@ -67,6 +67,8 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
     /* We need a 2nd scratch reg for several operations on AArch32 and AArch64 only. */
     reg_id_t reg2 = DR_REG_NULL;
 
+    drmgr_disable_auto_predication(drcontext, bb);
```

Comment:
BTW, if enough samples end up disabling, should we consider making this opt-in instead of opt-out?  I guess it still seems like wanting to predicate is more common than not.